### PR TITLE
Updating FFI-YAJL

### PR DESCRIPTION
**Pull request by johnmccrae**
## Description
testing version update for ffi-yajl

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] If `Gemfile.lock` has changed, I have used `--conservative` to do it and included the full output in the Description above.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: chef.gemspec [hunk 1]
```diff
@@ -39,7 +39,8 @@ Gem::Specification.new do |s|
   s.add_dependency "inspec-core", "~> 5.22.36"
 
   s.add_dependency "ffi", "~> 1.15.5"
-  s.add_dependency "ffi-yajl", "~> 2.2"
+  s.add_dependency "ffi-yajl", ">= 2.2", "< 4.0"
+  # s.add_dependency "ffi-yajl", "~> 2.2"
```
Comment (tpowell-progress) [inline chef.gemspec:43]: Go ahead and delete

**Comment by tpowell-progress** [comment]
@johnmccrae do you have the `bundle update` output?

**Review by tpowell-progress — CHANGES_REQUESTED** [review]
Clean up and bundle update output

File: chef.gemspec (hunk 1 above)
Comment (johnmccrae) [reply chef.gemspec]: done

**Review by tpowell-progress — APPROVED** [review]

**Pull request merged**